settings.ts

```diff
@@ -0,0 +1,32 @@
+/** Constants */
+export const YOUTUBE_BASE_URL = "youtube.com";
+export const DEFAULT_TEMPLATE = "---\ndate: {{Date}}\n---\n# {{Title}}\n![]({{ImageURL}})\n## Description:\n{{Description}}\n-> [Youtube video Link]({{PodcastURL}})\n\n## Summary:\n";
```

Comment:
Can you change `PodcastURL` to `VideoUrl` or `YoutubeVideoURL` here and everywhere else? As it will now be visible to users, it is good to be more clear about it:) 

I initially used the code from [another repo](https://github.com/marcjulianschwarz/obsidian-podcast-note) and it seems that I have forgotten to change it.

Reply:
Updated `PodcastURL` to `VideoUrl` in [`561577216f92dc2246cbe2967e97981909d59e24`](https://github.com/ozdemir08/youtube-video-summarizer/pull/9/commits/561577216f92dc2246cbe2967e97981909d59e24)